todo_app/templates/index.html

```diff
@@ -12,7 +12,47 @@ <h1 class="display-4">To-Do App</h1>
       <h2>Items</h2>
       <ul class="list-group mb-4">
         <!-- List the todo items here -->
+        {% for item in items %}
+        <li>
+        {{ item.id }} - {{ item.status }} - {{ item.title }}
+        </li>
+        {% endfor %}
       </ul>
     </div>
   </div>
+
+   <!-- form added for add item -->
+    <div class="row">
+      <div class="col-md-12">
+      <h2>Add Item</h2>
+            <form action = "/additem" method = "POST">
+	           <p>Title Name <input type = "text" name = "title" /></p>
+	           <p><input type = "submit" value = "Additem" /></p>
+      		</form>
+      	</div>
+      </div>
+
+     <!-- form added for mark complete item -->
+          <div class="row">
+	        <div class="col-md-12">
+	        <h2>Update Item</h2>
+	              <form action = "/updateitem" method = "POST">
+	  	           <p>ID from above list to be updated <input type = "text" name = "ID" /></p>
+	  	           <p><input type = "submit" value = "Updateitem" /></p>
+	        		</form>
```

Comment:
Optional improvement, can you add a button to each list element above that marks that item as complete, rather than asking for an id?